src/Parser/Priest/Holy/Modules/Talents/45/CosmicRipple.js

```diff
@@ -38,17 +39,20 @@ class CosmicRipple extends Analyzer {
   statistic() {
     return (
 
-      <StatisticBox
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
```

Comment:
This is a talent, but uses a TraitStatisticBox -- same goes for almost all the other talents aswell

Reply:
The components would be the same. I can create a carbon copy of the existing one I can create one.